applications/StructuralMechanicsApplication/custom_elements/base_solid_element.h

```diff
@@ -954,14 +954,14 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) BaseSolidElement
      * global to local coordinates
      * @param rValues The constitutive laws parameters
      */
-    void RotateToLocalAxes(ConstitutiveLaw::Parameters &rValues);
+    void RotateToLocalAxes(ConstitutiveLaw::Parameters &rValues, KinematicVariables& rThisKinematicVariables);
 
     /**
      * @brief This method rotates the F or strain according to local axis from
      * local de global
      * @param rValues The constitutive laws parameters
      */
-    void RotateToGlobalAxes(ConstitutiveLaw::Parameters &rValues);
+    void RotateToGlobalAxes(ConstitutiveLaw::Parameters &rValues, KinematicVariables& rThisKinematicVariables);
```

Comment:
```suggestion
    void RotateToGlobalAxes(
        ConstitutiveLaw::Parameters &rValues,
        KinematicVariables& rThisKinematicVariables);
```
Minor style guide suggestion.

Reply:
A prioti the ); should go to the next line